### DISCUSSION TITLE
How to upload csv data into existing table

### DISCUSSION THREAD

**Discussion by santoshyadavdev** [supabase/supabase#1508] [Questions] [answered]
I have created a table and now want to upload the existing data which I have in a csv file.

What are the ways to achieve it, I can see there is an option called **Data imports** where I can access this option.
![image](https://user-images.githubusercontent.com/11923975/117581504-b9d5e180-b11a-11eb-92ef-0a99f6947f15.png)

**Comment by easy-street** [comment, marked as answer]
The data imports feature is available when adding a new table: New table -> Add existing content to new table -> Upload CSV.

Most [PostgreSQL clients](https://supabase.io/docs/guides/local-development#before-you-begin) support importing from CSV into an existing table. Connect with the database connection string available in the dashboard at Settings -> Database.

**Reply by santoshyadavdev** [reply]
Perfect thanks 😊 though it would be great to have an upload data feature for the existing table too from the UI.
